Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py

```diff
@@ -944,8 +944,11 @@ def test_return_command_results(self):
             outputs=None,
             indicators=[ip]
         )
+        context = results.to_context()
+        del context['IndicatorTimeline']
```

Comment:
I think it will be more clear to 
- Either not add these fields if not provided in the `to_context()`, this way they will not appear here
- Add these fields to the expected results and not delete from the context